src/controllers/Login-Controller.coffee

```diff
@@ -87,6 +91,27 @@ class Login_Controller
   render_Page: (page, view_Model)=>
     @.res.send @.jade_Service.render_Jade_File page, view_Model
 
+  webServiceResponse: (methodName,Token,callback)->
+    options =
+      method: 'post',
+      body: {},
+      json: true,
+      headers: {'Cookie':'Session='+Token}
+      url: @.webServices + '/' + methodName
+    request options, (error, response)=>
+      if error
+        logger?.info ('Could not connect with TM 3.5 server')
+        callback null
+      callback response?.body?.d
```

Comment:
There is a bug here where if error is true then the callback is going to be called twice in this case it will make it easier to read if you use
```
if error
        logger?.info ('Could not connect with TM 3.5 server')
        callback null
else
        callback response?.body?.d
```